client/services/cardKeyService.js

```diff
@@ -8,14 +8,11 @@ const createMessage = (command, data) => {
   return new Uint8Array([0xaa, 0x00, ...payload, 0xbb]).buffer;
 };
 
-const convertUID = data => {
-  const reversed = data
+const convertUID = data =>
+  data
     .join('')
     .match(/.{1,2}/g)
```

Comment:
This also reverses the UID, which the electron app doesn't. I'd suggest reverting to the old UID (see earlier comment).